goboscript/res/grammar.lark

```diff
@@ -95,10 +95,14 @@ macrovar: MACROVAR
   | "backdrop" "num" "of" "stage" -> backdrop_num_of_stage
   | "backdrop" "name" "of" "stage" -> backdrop_name_of_stage
   | "volume" "of" "stage" -> volume_of_stage
+  | NAME "of" "stage" -> var_of_stage
   | "costume" "num" "of" STRING -> costume_num_of
   | "costume" "name" "of" STRING -> costume_name_of
   | "x" "pos" "of" STRING -> x_pos_of
   | "y" "pos" "of" STRING -> y_pos_of
+  | "direction" "of" STRING -> direction_of
```

Comment:
This is not required as the `var_of` rule can already handle this, the only benefit of explicitly writing this could be that the error-reporting on syntax errors would suggest these words, but that would only trigger when the syntax is malformed, not on spelling mistakes such as writing  `volum of "sprite"` because currently we aren't checking if the variables in `var_of` actually exist or not.